### PR TITLE
Merge Member into User

### LINKED ISSUE #13: Member and User seem to be the same thing
Can we merge them?

### PR DESCRIPTION
**Pull request by ChrisTheBaron**
Fixes #13

### CODE REVIEW

**Comment by CaptainHayashi** [comment]
I'd have put the merged stuff in `user.go` rather than `member.go` myself

**Comment by LordAro** [comment]
Agree, file now needs renaming

**Comment by ChrisTheBaron** [comment]
@LordAro, @CaptainHayashi review pls

File: lists.go [hunk 1]
```diff
@@ -28,17 +28,17 @@ func (s *Session) GetAllLists() ([]List, error) {
 	return lists, nil
 }
 
-// GetMembers retrieves all members subscribed to a given mailing list.
+// GetUsers retrieves all users subscribed to a given mailing list.
 // This consumes one API request.
-func (s *Session) GetMembers(l *List) ([]Member, error) {
+func (s *Session) GetUsers(l *List) ([]User, error) {
```
Comment (LordAro) [inline lists.go]: I presume 2016-site (or wherever is aware of the API change?

Comment (ChrisTheBaron) [inline lists.go]: It will be when someone tries to run `go build` on it.

**Pull request merged**